src/index.ts

```diff
@@ -205,6 +204,16 @@ class SOOSDASTAnalysis {
       nargs: "*",
     });
 
+    analysisArgumentParser.addEnumArgument(
```

Comment:
no merge or update from main until we do the fix itself.

Reply:
we can hopefully release this with the logging change and debug info